src/components/sbb-toast/sbb-toast.tsx

```diff
@@ -150,17 +158,40 @@ export class SbbToast implements ComponentInterface {
     this.willClose.emit();
   }
 
-  @Method() public async getState(): Promise<SbbOverlayState> {
-    return this._state;
+  @Listen('click') public async onClick(ev: Event): Promise<void> {
```

Comment:
completely nitpicky comment, but so far we used `evt` and `event`. `ev` would be new (not completely, it's in autocomplete as well :P . But I let it up to you :D
```suggestion
  @Listen('click') public async onClick(event: Event): Promise<void> {
```

Reply:
I'll change those, no problem :)